src/apps/VirtualApp/settings/Settings.js

```diff
@@ -55,19 +56,26 @@ const roomDescriptionById = new Map();
 
 const Settings = (props) => {
   const {classes} = useStyles();
+  const [shidurMuted, setShidurMuted] = useState(JanusStream.audioElement.muted)
 
   const {t} = useTranslation();
   const {palette: {background: {paper}}} = useTheme();
   const {isDark, toggleTheme} = useContext(ThemeContext);
 
-  const {audio, video, rooms, isAudioMode, isGroup, initClient, selectedRoom, selectRoom, setAudioDevice, setVideoDevice, settingsChange, audioModeChange, handleGroupChange, videoLength, videoSettings, audioDevice = audio.devices[0]?.deviceId, videoDevice = video?.devices[0]?.deviceId, userDisplay, delay, startLocalMedia, stopLocalMedia, cammuted,} = props;
+  const {audio, video, rooms, isAudioMode, isGroup, initClient, selectedRoom, selectRoom, setAudioDevice, setVideoDevice, settingsChange, audioModeChange, handleGroupChange, videoLength, videoSettings, audioDevice = audio.devices[0]?.deviceId, videoDevice = video?.devices[0]?.deviceId, userDisplay, delay, startLocalMedia, stopLocalMedia, cammuted, toggleUsersDisplays, hideDisplays,} = props;
 
   useEffect(() => {
     for (const r of rooms) {
       roomDescriptionById.set(r.room, r);
     }
   }, [rooms]);
 
+  const handleMute = () => {
+    JanusStream.audioElement.muted = !shidurMuted
+    JanusStream.trlAudioElement.muted = !shidurMuted
```

Comment:
not sure about trlAudioElement

Reply:
I'm not inderstand what the problem you try to solve